src/sidebar/filter/persona.js

```diff
@@ -30,7 +30,22 @@ function username(user) {
   return account.username;
 }
 
+/**
+ * Returns true if the authority of an account ID is Hypothes.is.
+ */
+function isHAuthority(user) {
+  var account = parseAccountID(user);
+  if (!account) {
```

Comment:
You can simplify this to `return account && account.provider === <the Hypothesis authority>`